hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneClientMultipartUploadV1.java

```diff
@@ -264,12 +274,232 @@ public void testUploadPartOverrideWithRatis() throws IOException {
     commitUploadPartInfo = ozoneOutputStream
             .getCommitUploadPartInfo();
 
-    assertNotNull(commitUploadPartInfo);
-    assertNotNull(commitUploadPartInfo.getPartName());
+    Assert.assertNotNull(commitUploadPartInfo);
+    Assert.assertNotNull(commitUploadPartInfo.getPartName());
 
     // PartName should be different from old part Name.
-    assertNotEquals("Part names should be different", partName,
+    Assert.assertNotEquals("Part names should be different", partName,
             commitUploadPartInfo.getPartName());
   }
 
+  @Test
+  public void testMultipartUploadWithPartsLessThanMinSize() throws Exception {
```

Comment:
Can we additionally add one another case for OMException.ResultCodes.INVALID_PART_ORDER? I didn't this case was covered in this unit test.

Reply:
Sure. As `bucket.completeMultipartUpload(keyName, uploadID, partsMap); ` used hashmap to store parts, which will always be in sorted order. Inorder to make part list with incorrect order, I have added UT `testInvalidPartOrderError` in `TestS3MultipartUploadCompleteRequest` class.